cmd/api/main.go

```diff
@@ -86,28 +89,30 @@ func NewServer(k8sClient kubernetes.Interface, controller routers.Controller, ca
 }
 
 func main() {
-	log.Printf("Starting KubeArchive API with version '%s', commit SHA '%s', built '%s'", version, commit, date)
+	slog.Info("Starting KubeArchive API", "version", version, "commit", commit, "built", date)
 	err := observability.Start(otelServiceName)
 	if err != nil {
-		log.Printf("Could not start opentelemetry: %s", err)
+		slog.Info("Could not start opentelemetry:", "error", err.Error())
```

Comment:
Why does this message have a colon?  None of the others seem to.

Reply:
No reason, I just forgot to remove this one.